tla-io/src/main/scala/at/forsyte/apalache/io/ReportGenerator.scala

```diff
@@ -0,0 +1,71 @@
+package at.forsyte.apalache.io
+
+import java.io.File
+import java.util.regex.Matcher
+import scala.io.Source
+
+object ReportGenerator {
+  private val reportFile = "BugReport.md"
+  private val reportTemplate = "issueTemplate.md"
+
+  private object tags {
+    val spec = "__SPEC"
+    val cmd = "__CMD"
+    val log = "__LOG"
+    val version = "__VER"
+    val os = "__OS"
+    val jdk = "__JDK"
+  }
+
+  private def readSrcIntoString(src: Source): String =
+    try src.mkString.trim
+    finally src.close()
+
+  private def readFileIntoString(file: File): String =
+    readSrcIntoString(Source.fromFile(file))
+
+  private def readContentsOfFileInRunDir(filename: String): String = OutputManager.runDirPathOpt
+    .map { runDir =>
+      readFileIntoString(new File(runDir.toFile, filename))
+    }
+    .getOrElse("")
+
+  // we drop the filename, since the path is local
+  def getCMD(): String =
+    readContentsOfFileInRunDir(OutputManager.Names.RunFile).split(" ").dropRight(1).mkString(" ")
```

Comment:
I don't understand the reason fro dropping the filename from the command. Does it do any harm to include it?

Reply:
No harm, but it's redundant information, since it won't help the person reading the issue, as you can't resolve, e.g. `C:\Users\User\Documents\Specs\xyz.tla` on your machine.

Reply:
But we could hit errors due to file names or something right? My rule of thumb is generally something like: don't do extra work to remove information that might later be useful, unless it should be a secret or is causing harm.

Reply:
We shouldn't read this from a file at all, actually. It is available in memory the whole time, so let's just get this data from: https://github.com/informalsystems/apalache/blob/2b736c9ffd874d583de0c1e57ecd100bc251b517/mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/General.scala#L43-L44

Reply:
Following up here: We don't need to read this value from a file, cause it's already stored in memory in the the CLI parser instance.